Running total off by one patch

nimmicus opened the pull request:
Running total is always off by one. The code did not account for empty first and last names. The initial empty first and last name string gets added to the full_name list and all others are dropped. This skews the total running count by one.

initstring commented on linkedin2username.py:
```diff
@@ -551,6 +551,12 @@ def scrape_info(session, company_id, staff_count, args):
             for first, last in zip(first_name, last_name):
                 full_name = first + ' ' + last
+
+		# Off-By-One Running Total Patch: Ensures that a blank first and 
+                # last name are not added to the list of full names.
```
```suggestion
                # Off-By-One Running Total Patch: Ensures that a blank first and 
                # last name are not added to the list of full names.
```
I think this fixes the indent (one line used tabs)

initstring commented:
Hi @nimmicus ,

Thanks so much for catching this bug! The indentation looked off in one of the comments. I only viewed in the browser, so maybe it was me. Could you double check and then I will merge?

Thanks!!!

nimmicus commented:
Hi @initstring:

You are correct, the first comment line I added has tabs instead of spaces. I corrected and pushed the change. Sorry about that. Tabbing is a habit.

Also, as others have said, thank you for sharing this tool with us.

initstring commented:
Looks good! Thanks so much for taking the time to contribute. Have a great day! :)
